cmake_file_api/kinds/cache/v2.py

```diff
@@ -17,15 +17,15 @@ class CacheEntryType(Enum):
     TYPE_UNINITIALIZED = "UNINITIALIZED"
 
 
-class CacheEntryProperty(object):
+class CacheEntryProperty:
     __slots__ = ("name", "value")
 
     def __init__(self, name: str, value: str):
         self.name = name
         self.value = value
 
     @classmethod
-    def from_dict(cls, dikt: Dict) -> "CacheEntryProperty":
+    def from_dict(cls, dikt: dict) -> "CacheEntryProperty":
```

Comment:
I think it's now possible to remove the unused typing imports?

Reply:
done